common/src/main/java/com/wynntils/models/character/type/ClassType.java

```diff
@@ -1,27 +1,31 @@
 /*
- * Copyright © Wynntils 2022-2024.
+ * Copyright © Wynntils 2022-2025.
  * This file is released under LGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.models.character.type;
 
+import com.wynntils.utils.type.Pair;
+
 public enum ClassType {
-    MAGE("Mage", "Dark Wizard", 1),
-    ARCHER("Archer", "Hunter", 2),
-    WARRIOR("Warrior", "Knight", 3),
-    ASSASSIN("Assassin", "Ninja", 4),
-    SHAMAN("Shaman", "Skyseer", 5),
+    MAGE("Mage", "Dark Wizard", 1, Pair.of("\uE002", "\uE007")),
+    ARCHER("Archer", "Hunter", 2, Pair.of("\uE000", "\uE005")),
+    WARRIOR("Warrior", "Knight", 3, Pair.of("\uE004", "\uE009")),
+    ASSASSIN("Assassin", "Ninja", 4, Pair.of("\uE001", "\uE006")),
+    SHAMAN("Shaman", "Skyseer", 5, Pair.of("\uE003", "\uE008")),
 
     // This represents the class selection menu, or the generic spell
-    NONE("none", "none", 0);
+    NONE("none", "none", 0, Pair.of("", ""));
 
     private final String name;
     private final String reskinnedName;
     private final int encodingId;
+    private final Pair<String, String> cardCharactersPair;
```

Comment:
It is not intuitive enough that this is a pair because there is a normal and a reskinned character. (At least that is what I think this is, but even for me it is not clear). Try to find a better variable name or at least comment on it's purpose.